src/SurfaceFluxes.jl

```diff
@@ -344,11 +344,11 @@ function obukhov_length end
 
 obukhov_length(sfc::SurfaceFluxConditions) = sfc.L_MO
 
-function non_zero_lmo(L_MO::FT) where {FT}
-    if L_MO == FT(0)
-        return L_MO + sqrt(eps(FT))
+function non_zero(value::FT) where {FT}
+    if value == FT(0)
```

Comment:
Just to note, we should probably replace this with a numerically better comparison. For example:

```julia
julia> Float64(1e-304) == Float64(0)
false
```
I thought @tapios suggested to use `abs(value) < eps(FT)` or `abs(value) < sqrt(eps(FT))`? This is probably more numerically sensible in terms of avoiding division by zero.

Reply:
Actually, it also get protected...
```
julia> non_zero(1e-304)
1.4901161193847656e-8
```

Reply:
Or, are you suggesting something like
```
function non_zero(value::FT) where {FT}
           if abs(value) < eps(FT)
               return value + sqrt(eps(FT))
           else
               return value 
           end
       end
```

Reply:
Ah, good point. I didn't realize that we're just always adding eps in the else case. Yeah, what you wrote is what I was suggesting.